src/app/features/location/geo.service.ts

```diff
@@ -69,14 +69,28 @@ export class GeoService {
   private reformatDisplayName(result: OpenStreetMapsSearchResult): GeoResult {
     const addr = result?.address;
     if (addr) {
-      const city = addr.city ?? addr.town;
+      const city = addr.city ?? addr.town ?? "";
+      let street = addr.road ? addr.road : "";
```

Comment:
a shorter way of writing this:
```suggestion
      let street = addr.road ?? "";
```

(see https://angular.love/double-question-marks-typescript-3-7-nullish-coalescing)